packages/design-system/src/styles/components/_Button.scss

```diff
@@ -135,27 +139,28 @@
     border-color: rgba($border-color-inverse, 0.8);
```

Comment:
I'm not actually sure why `ds-c-button--active` is included in this statement, when we have [this](https://github.com/CMSgov/design-system/pull/724/files#diff-ebbcedeff6d7cdd3db8f187464ffad26R153) a few lines down. It might be a bug, and we can probably remove those active selectors

Reply:
Ah good catch, I'm thinking we remove this block from below 
```
&:active,
  &.ds-c-button--active {
    border-color: rgba($border-color-inverse, 0.6);
    color: rgba($color-base-inverse, 0.6);
  }
```

Reply:
I don't think thats the best approach, as we want the `active` style to be different than hover. I think we want to do the opposite and remove line 136 and 138